ciongke/cmd/hook/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hook
+  labels:
+    app: hook
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: hook
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: hook
+        image: "gcr.io/k8s-ci-on-gke/hook:0.0"
```

Comment:
Right now I'm just storing the YAML in the makefile, which is pretty ugly, but I think it actually makes sense.